Bottom: db292631e2a9947ae48977d3ab4325c7e2702d8d
Top:    7217448eba5be8a04103c0fd1ddebe65f397a57e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:06 +0000

Remove left over debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 128ae26..389c255 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1251,8 +1251,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 #endif
                                 copper_appearance,
                                 NULL));
-
-      //break;
     }
